README.md

```diff
@@ -14,7 +14,9 @@ SIDECAR_SECRET_PATH: optional defaults to  '/secrets'
 Your kubernetes cluster will also requires a few objects in order for this
 to work.  A token or an pemfile (VAULT_AUTH_FILE) will need to be created
 in vault, then the secret object will need to be created.  The contents
-of the secret must be base64 encoded, and cannot include EOF.  See:
+of the secret must be base64 encoded, and cannot include EOF
+**(note that the secrets in this repo are for testing only)**.    
```

Comment:
could also read this as "real secrets, do not use except for testing"

"secrets are made up/invalid values and won't work except for testing"